cookbooks/aws-parallelcluster-common/libraries/helpers.rb

```diff
@@ -19,3 +19,7 @@ def x86?
 def arm_instance?
   node['kernel']['machine'] == 'aarch64'
 end
+
+def efa_supported?
+  !arm_instance? || !default['cluster']['efa']['unsupported_aarch64_oses'].include?(node['cluster']['base_os'])
+end
```

Comment:
This one should also be moved to networking library.

Reply:
done